base.py

```diff
@@ -897,10 +1165,31 @@ class MBDynPlotVar_UL_List(bpy.types.UIList):
     def draw_item(self, context, layout, data, item, icon, active_data, active_propname, index):
         layout.label(item.name)
 # -----------------------------------------------------------
-# end of MBDynNodes_UL_List class
-    
+# end of MBDynPLotVar_UL_List class
+
+class MBDynEnvVar_UL_List(bpy.types.UIList):
```

Comment:
I don't know if this is the most logical place for this panel... Maybe it is better to move it in the `TOOLS ` region, near the operators that allow the user to control the MBDyn simulation from the Blender interface?